core/drivers/pm/sam/at91_pm.c

```diff
@@ -280,6 +280,16 @@ static TEE_Result at91_write_backup_data(void)
 	return TEE_SUCCESS;
 }
 
+static void at91_pm_change_state(enum pm_op op)
+{
+	int suspend_type = soc_pm.mode == AT91_PM_STANDBY ?
+			   PM_SUSPEND_STANDBY : PM_SUSPEND_TO_MEM;
+
+	uint32_t hint = SHIFT_U32(suspend_type, PM_HINT_SUSPEND_TYPE_SHIFT);
+
+	pm_change_state(op, hint);
+}
+
```

Comment:
Avoid the ternary operator and declare and initialize all variables at the beginning.

```C
static void at91_pm_change_state(enum pm_op op)
{
	int type = 0;
	uint32_t hint = 0;
 
	if (soc_pm.mode == AT91_PM_STANDBY)
		type = PM_SUSPEND_STANDBY;
	else
		type = PM_SUSPEND_TO_MEM;

	hint = SHIFT_U32(type, PM_HINT_SUSPEND_TYPE_SHIFT);

	pm_change_state(op, hint);
}
```